torch/testing/_internal/common_methods_invocations.py

```diff
@@ -7307,6 +7331,14 @@ def generate_std_var_kwargs(t: torch.Tensor, **kwargs):
            dtypes=floating_and_complex_types(),
            sample_inputs_func=sample_inputs_linalg_slogdet,
            decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack]),
+    OpInfo('linalg.trace',
+           aten_name='linalg_trace',
+           supports_out=False,
```

Comment:
Now that we are at it, it would be good to make this operation support out, as it would take very little effort to do so.

Reply:
Do you think should `out` be a keyword only argument or not?
Other PyTorch functions can take `out` as a positional argument, but `torch.linarg.trace(A, out, *, offset)` is confusing because it is not compatible with `numpy.trace(A, offset)`.

Reply:
`out` is always kwonly and the last output in PyTorch, as such it should be `torch.linarg.trace(A, *, offset=0, out=None)`